cdap-standalone/src/main/resources/cdap-site.xml

```diff
@@ -197,4 +197,12 @@
     </description>
   </property>
 
+  <property>
+    <name>kerberos.auth.enabled</name>
+    <value>false</value>
+    <description>
+      Disable Kerberos authentication in standalone.
```

Comment:
Description isn't supposed to be tied to the specific value we're configuring, right?

Probably just copy the description from cdap-default.xml - that's what the other properties in here seem to be doing.